<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Producto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Proveedor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddProduct">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEnableEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>habilitar/deshabilitar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditProduct">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSupplier">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
